src/components/Gouvernance/Membre/DetailsMembre.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { fireEvent, within, screen, render } from '@testing-library/react'
+
+import Gouvernance from '../Gouvernance'
+import { gouvernancePresenter } from '@/presenters/gouvernancePresenter'
+import { gouvernanceReadModelFactory } from '@/use-cases/testHelper'
+
+describe('membres', () => {
+  it('quand je clique sur un membres, alors un drawer s’ouvre avec les détails du membre', () => {
+  // GIVEN
+    const gouvernanceViewModel = gouvernancePresenter(gouvernanceReadModelFactory())
+    render(<Gouvernance gouvernanceViewModel={gouvernanceViewModel} />)
+
+    // WHEN
+    const membre = screen.getByRole('button', { name: 'Préfecture du Rhône' })
+    fireEvent.click(membre)
+
+    // THEN
+    const drawer = screen.getByRole('dialog')
```

Comment:
```suggestion
    const drawer = screen.getByRole('dialog', { name: 'Préfecture du Rhône' })
```
Ce qui permet de dire que le drawer est bien lié à son titre.
Je m'aperçois que ce n'est pas cirrect dans les autres tests donc peux tu faire ce refactoring dans tous les autres stp ? Il y en a 4